extensions/mssql/src/objectExplorerNodeProvider/fileSources.ts

```diff
@@ -187,18 +188,23 @@ export class HdfsFileSource implements IFileSource {
 				reject(error);
 			});
 
+			let data: any[] = [];
 			if (maxBytes) {
 				remoteFileStream = remoteFileStream.pipe(meter(maxBytes));
 				remoteFileStream.on('error', (err) => {
 					error = <HdfsError>err;
 					if (error.message.includes('Stream exceeded specified max')) {
-						error.message = localize('maxSizeReached', 'File exceeds max size of ${0}', bytes(maxBytes));
+						// We have data > maxbytes, show we're truncating
+						let previewNote: string = '########################### NOTICE: This file has been truncated at '+ bytes(maxBytes) +' for preview ########################### \r\n';
```

Comment:
2 things:
1. Please put a #### string above/below the truncation notice so it takes up more space per Vicky's suggestion
```
#########################################################################################################
####################### NOTICE: This file has been truncated at 30 MB for preview ##################################
#########################################################################################################
```
2. The truncation section should be localized. Therefore, you'll want to 
    - Get the localized string & find its length
    - Use this to output the right number of #####s to show on the lines above/below this

Reply:
Sure thing!